pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java

```diff
@@ -226,20 +242,83 @@ public static String toString(final Class<?> clazz, final Object... args) {
      * @return the input stream
      */
     public static InputStream getInputStreamFromName(String name) {
-        if (name.startsWith(RESOURCE_PREFIX)) {
-            String path = name.substring(RESOURCE_PREFIX.length());
-            if (!path.startsWith("/")) {
-                path = "/" + path;
-            }
-            return CommonHelper.class.getResourceAsStream(path);
-        } else {
-            try {
-                return new FileInputStream(name);
-            } catch (FileNotFoundException e) {
-                throw new TechnicalException(e);
-            }
-        }
-    }
+		int prefixEnd = name.indexOf(":");
+		String prefix = null;
+		String path = name;
+		if (prefixEnd != -1) {
+			prefix = name.substring(0, prefixEnd);
+			path = name.substring(prefixEnd + 1);
+		}
+		if (prefix == null) {
```

Comment:
I added the if null for findbugs which told me that the prefix maybe null.. I'll take it out and see if travis will build